src/EditorFeatures/Core.Wpf/Completion/Presentation/RoslynCompletionSet.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.VisualStudio.Language.Intellisense;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
+using RoslynCompletion = Microsoft.CodeAnalysis.Completion;
 using VSCompletion = Microsoft.VisualStudio.Language.Intellisense.Completion;
 using CompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
```

Comment:
This line can be removed since you're using `RoslynCompletion.CompletionItem` below. Also remove this using from other files